### PR TITLE
add component to show other payment due accounts

### PR DESCRIPTION
**Pull request by deepan83**
- Add other payment accounts component
- Show sim bill balance when available

### CODE REVIEW

File: addon/make-a-payment/confirmation/next-payment-action/component.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import Ember from 'ember';
+import { computed } from 'ember-decorators/object';
+
+export default class NextPaymentAction extends Ember.Component {
+
+  @computed('energyAccounts', 'currentEnergyAccount')
+  get otherEnergyAccountsInDebit () {
+    const energyAccounts = this.get('energyAccounts') || [],
+      currentEnergyAccount = this.get('currentEnergyAccount');
```
Comment (Leooo) [inline addon/make-a-payment/confirmation/next-payment-action/component.js]: cleaner as a property of `energyAccount`? (still named `otherEnergyAccountsInDebit`), that could reuse a premise 'energyAccountsInDebit` property and filter out `this` ?

Comment (deepan83) [reply addon/make-a-payment/confirmation/next-payment-action/component.js]: dont want to move all state to energyAccounts until required

Comment (deepan83) [reply addon/make-a-payment/confirmation/next-payment-action/component.js]: if we find there is reuse of this property, then we can move it there

Comment (deepan83) [reply addon/make-a-payment/confirmation/next-payment-action/component.js]: at the moment, it is just this component that is interested in this

Comment (Leooo) [reply addon/make-a-payment/confirmation/next-payment-action/component.js]: but how will someone wanting to reuse the logic be able to find it in this repo ? ;)

Comment (Leooo) [reply addon/make-a-payment/confirmation/next-payment-action/component.js]: Can we add unit tests for this? It is a complicated property.

**Pull request merged**